### PR TITLE
Check the previous cost before sending an alert

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description
Check the remaining cost in the last email before sending the budget email. Reduced the aggressiveness from 10 -> 15 days.

### CODE REVIEW

File: cloud_governance/cloud_resource_orchestration/common/abstract_monitor_tickets.py [hunk 1]
```diff
@@ -192,13 +192,13 @@ def _monitor_ticket_duration(self, ticket_id: str, region_name: str, duration: i
...
-    def get_budget_exceed_alert_times(self, user: str):
+    def get_budget_exceed_alert_times(self, user: str, remaining_budget: int):
         """
-        This method returns the number of times alerts send to an user
+        This method returns the number of times alerts send to a user
         :return:
         """
         current_date = datetime.now(timezone.utc).date()
-        start_date = current_date - timedelta(days=10)
+        start_date = current_date - timedelta(days=15)
```
Comment (ebattat) [inline cloud_governance/cloud_resource_orchestration/common/abstract_monitor_tickets.py:201]: Why we need to increase the days to 15 ?

Comment (athiruma) [reply cloud_governance/cloud_resource_orchestration/common/abstract_monitor_tickets.py:201]: To reduce the mailing aggressiveness, send mail every 2 weeks.

Comment (ebattat) [reply cloud_governance/cloud_resource_orchestration/common/abstract_monitor_tickets.py:201]: ok

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**